interface/java_binding/src/test/java/org/khronos/ktx/test/KtxTexture2Test.java

```diff
@@ -599,4 +599,37 @@ public void testSupercompressionZLIB() throws IOException {
 
         t.destroy();
     }
+
+    @Test
```

Comment:
Please add a comment similar to what you wrote in the PR description about this test.

Reply:
That certainly makes sense (otherwise, people might wonder what the purpose of the test is). 

(An aside: In another context, I automated this kind of test, [using some reflection magic](https://github.com/jcuda/jcuda/blob/master/JCudaJava/src/test/java/jcuda/test/BasicBindingTest.java) - but here, we only have ~20 `native` functions, whereas for JCuda, it was about (literally) thousands of `native` methods in dozens of classes...)